pkg/util/containers/types.go

```diff
@@ -93,9 +94,21 @@ type NetworkDestination struct {
 	Mask      uint64
 }
 
+// This is absolutely not the place for this interface
+// TODO: Fixme, put me in a package that is shared between the providers and docker
+type DockerApiWrapper interface {
```

Comment:
Why do we need this interface? `DockerUtil` is a global singleton, should be accessible from Provider whenever we want?